backend/bootstrap_handler.py

```diff
@@ -0,0 +1,19 @@
+"""
+The handler of this module will be called once upon every deployment
+"""
+
+import logging
+import os
+
+from dataall.base.db import get_engine
+from dataall.core.permissions.db import save_permissions_with_tenant
+
+logger = logging.getLogger()
+logger.setLevel(os.environ.get('LOG_LEVEL', 'INFO'))
+log = logging.getLogger(__name__)
+
+
+def save_permissions(event, context):
```

Comment:
In real deployment we have a trigger Lambda that executes this handler before the api_handler. For local deployment, we still keep the save_permissions_with_tenant in the local_graphql_server. That is alright right?

Reply:
I deliberately didn't touch the local_server because it execute the save_permissions upon starting not on the first request, hence it makes no difference.